CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - google.golang.org/grpc/otelgrpc: Avoid getting a new Tracer for every RPC.
 - google.golang.org/grpc/otelgrpc: Deprecate Inject and Extract public funcs.
 - google.golang.org/grpc/otelgrpc: Conditionally compute message size for tracing events using proto v2 API rather than legacy v1 API.
+- Add trace propagation support to `instrumentation/github.com/aws/aws-sdk-go-v2/otelaws` (#2856).
```

Comment:
```suggestion
- Add trace context propagation support to `instrumentation/github.com/aws/aws-sdk-go-v2/otelaws` (#2856).
```

This item should go in an `### Added` heading.

Reply:
Done. Thanks for reviewing.